packages/edge-login-ui-rn/src/components/abSpecific/AccountInfoComponent.js

```diff
@@ -149,6 +149,7 @@ class AccountInfoComponent extends React.Component<Props, State> {
     return (
       <View style={style.top}>
         <TextAndIconButton
+          testID={this.props.testID}
```

Comment:
This was not added to the props above. Flow should have caught this in the pre-commit hook.